upup/pkg/fi/cloudup/apply_cluster.go

```diff
@@ -72,7 +72,7 @@ const (
 )
 
 var (
-	// AlphaAllowDO is a feature flag that gates DigitalOcean support while it is alpha
+	// AlphaAllowDO is a feature flag that gates cloudprovider support while it is alpha
 	AlphaAllowDO = featureflag.New("AlphaAllowDO", featureflag.Bool(false))
```

Comment:
We can just remove this feature flag altogether right? Below was the only place its used so enabling DO support by default should involve removing this FF.

Reply:
I know there was another PR on Azure support to DO, so was not sure if it would get into the Alpha state. I can remove this flag if you think we should.

Reply:
I think if we're removing any use of the flag, effectively making its default behavior to being enabled, then we should just remove the flag altogether given that explicitly disabling the use of a cloud provider feature flag doesn't make sense.

Reply:
Removed.